core/consensus/authority/authority_manager_error.cpp

```diff
@@ -10,10 +10,10 @@ OUTCOME_CPP_DEFINE_CATEGORY(kagome::authority, AuthorityManagerError, e) {
   switch (e) {
     case E::UNKNOWN_ENGINE_ID:
       return "Unknown engine_id";
-    case E::WRONG_FINALISATION_ORDER:
-      return "Finalisation of block which isn't descendant of last finalised";
     case E::ORPHAN_BLOCK_OR_ALREADY_FINALISED:
       return "Block it not descendant of last finalised block";
+    case E::CAN_NOT_SAVE_STATE:
+      return "Can not save state";
```

Comment:
It would be great if it were possible to emit a more informative error.

Reply:
Particular reason will be written to the log. It is general error for failed saving case.